[2023-10-17T11:19:53.524+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-06-24T00:00:00+00:00 [queued]>
[2023-10-17T11:19:53.533+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-06-24T00:00:00+00:00 [queued]>
[2023-10-17T11:19:53.533+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-17T11:19:53.556+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-06-24 00:00:00+00:00
[2023-10-17T11:19:53.562+0000] {standard_task_runner.py:57} INFO - Started process 791 to run task
[2023-10-17T11:19:53.564+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-06-24T00:00:00+00:00', '--job-id', '179', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmpzjufnfrs']
[2023-10-17T11:19:53.565+0000] {standard_task_runner.py:85} INFO - Job 179: Subtask waiting_for_data
[2023-10-17T11:19:53.609+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-06-24T00:00:00+00:00 [running]> on host 238c1d4ff767
[2023-10-17T11:19:53.686+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-06-24T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-06-24T00:00:00+00:00'
[2023-10-17T11:19:53.696+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-17T11:19:53.697+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-17T11:19:53.697+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-17T11:19:53.697+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-17T11:19:53.706+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210624T000000, start_date=20231017T111953, end_date=20231017T111953
[2023-10-17T11:19:53.737+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-17T11:19:53.754+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-18T10:43:37.631+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-06-24T00:00:00+00:00 [queued]>
[2023-10-18T10:43:37.642+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-06-24T00:00:00+00:00 [queued]>
[2023-10-18T10:43:37.643+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-18T10:43:37.685+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-06-24 00:00:00+00:00
[2023-10-18T10:43:37.695+0000] {standard_task_runner.py:57} INFO - Started process 1754 to run task
[2023-10-18T10:43:37.698+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-06-24T00:00:00+00:00', '--job-id', '178', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmptu8tae0n']
[2023-10-18T10:43:37.699+0000] {standard_task_runner.py:85} INFO - Job 178: Subtask waiting_for_data
[2023-10-18T10:43:37.742+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-06-24T00:00:00+00:00 [running]> on host 2473b23ba069
[2023-10-18T10:43:37.803+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-06-24T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-06-24T00:00:00+00:00'
[2023-10-18T10:43:37.818+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/base.py", line 257, in execute
    raise e
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/base.py", line 239, in execute
    poke_return = self.poke(context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/filesystem.py", line 61, in poke
    hook = FSHook(self.fs_conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/hooks/filesystem.py", line 38, in __init__
    conn = self.get_connection(conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/hooks/base.py", line 72, in get_connection
    conn = Connection.get_connection_from_secrets(conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/connection.py", line 477, in get_connection_from_secrets
    raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")
airflow.exceptions.AirflowNotFoundException: The conn_id `fs_default` isn't defined
[2023-10-18T10:43:37.821+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210624T000000, start_date=20231018T104337, end_date=20231018T104337
[2023-10-18T10:43:37.831+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 178 for task waiting_for_data (The conn_id `fs_default` isn't defined; 1754)
[2023-10-18T10:43:37.871+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-10-18T10:43:37.896+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-18T11:43:59.039+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-06-24T00:00:00+00:00 [queued]>
[2023-10-18T11:43:59.054+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-06-24T00:00:00+00:00 [queued]>
[2023-10-18T11:43:59.054+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-18T11:43:59.074+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-06-24 00:00:00+00:00
[2023-10-18T11:43:59.083+0000] {standard_task_runner.py:57} INFO - Started process 1163 to run task
[2023-10-18T11:43:59.090+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-06-24T00:00:00+00:00', '--job-id', '309', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmp172242el']
[2023-10-18T11:43:59.091+0000] {standard_task_runner.py:85} INFO - Job 309: Subtask waiting_for_data
[2023-10-18T11:43:59.155+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-06-24T00:00:00+00:00 [running]> on host 0e68aa52dfc7
[2023-10-18T11:43:59.262+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-06-24T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-06-24T00:00:00+00:00'
[2023-10-18T11:43:59.277+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-18T11:43:59.278+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-18T11:43:59.278+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-18T11:43:59.278+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-18T11:43:59.292+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210624T000000, start_date=20231018T114359, end_date=20231018T114359
[2023-10-18T11:43:59.346+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-18T11:43:59.401+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
